ngraph/frontend/paddlepaddle/src/op/cumsum.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <node_context.hpp>
+
+#include "default_opset.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+NamedOutputs cumsum(const NodeContext& node) {
+    auto x = node.get_ng_input("X");
```

Comment:
Looks like we don't change `x` and other variables, so I think it can be const.

Reply:
Updated.